create_pr_quick_start_guide.rst

```diff
@@ -0,0 +1,244 @@
+*****************************
+Quick-start development guide
```

Comment:
Since this guide is about collections, maybe we should drop the ansible/ansible git stuff because it is not strictly necessary? Using the latest stable Ansible Core should be good enough. This will also help keep the guide a bit more focused.

I would also introduce venv here to keep the things contained and easy to remove later on.

Reply:
@tadeboro , thanks for the suggestions!
1)  Ansible-Core - removed
2) venv - i'd leave only really necessary things there to make the guide really quick:)